packages/liveblocks-react/src/factory.tsx

```diff
@@ -560,7 +569,7 @@ export function createRoomContext<
       [selector]
     );
 
-    const getServerSnapshot = React.useCallback((): Snapshot => null, []);
+    const getServerSnapshot = alwaysNull;
```

Comment:
Avoiding `useCallback()` wrappers in all of our hooks might actually be a nice pattern to use for a library (I would not do this pattern in application code). I think we could rewrite most of these `useCallback`s and use a stable function reference instead, similar to this change.

I only changed it for the `() => null` factory here, but I think it could be nice just so React doesn't have to add all those callback functions to its registry and compare them on every render, etc.